Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/common/use-success-message-effect.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from 'react';
+
+import { ISuccessMessageData, ISuccessMessageEffectParams } from '../../common/types';
```

Comment:
These types can be defined here no need to import them if they are going to be used only in this file.

Reply:
Done.